src/images/1x/Academia.svg

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Yeah this new image looks much better. Something about it seems odd on mobile but I'll bring this up with Dan. Thanks!

<img width="208" alt="Screen Shot 2020-06-02 at 4 04 47 PM" src="https://user-images.githubusercontent.com/9064401/83564556-e2a52e00-a4ea-11ea-8088-9b3556b378b8.png">